owtf.py

```diff
@@ -255,4 +255,9 @@ def main(args):
 
 
 if __name__ == "__main__":
+
+    if os.path.dirname(os.path.abspath(sys.argv[0])) == os.getcwd():
```

Comment:
Instead of this check, check if OWTF is ran from its directory/sub-directories. Check for the substring, @delta24 what is your view?